### ISSUE TITLE
Some GTK 3 apps don't get themed

### ISSUE THREAD

**Issue by An-Eagle** [i-mint/midnight#12]
Some specific GTK 3 apps are not getting affected by the theme, I believe these are all the core gnome apps, as I have found that gnome tweaks and gnome system-monitor don't work. All my other gtk 4 apps respect the theme, and nothing appears when launching these apps from the terminal.
I have tried other themes like flat remix and arc theme, and these function correctly

**Comment by i-mint** [comment]
This theme does not yet support the GTK-4.0 (Libadwaita), it should work well on GTK-3.0 including Gnome Tweaks and Gnome System Monitor

**Comment by An-Eagle** [comment]
> 



> This theme does not yet support the GTK-4.0 (Libadwaita), it should work well on GTK-3.0 including Gnome Tweaks and Gnome System Monitor

Yes, I know, but it does not seem to apply for me on gnome 44(and 43), for gnome tweaks and system monitor. Other themes do, though, and so do other gtk 3 applications

**Comment by An-Eagle** [comment]
Small correction, The theme only applies if I set my gnome to light theme, while it breaks in dark theme.

Renaming midnight/gtk-3.0/gtk.css to gtk-dark.css seems to fix it

**Comment by i-mint** [comment]
> Small correction, The theme only applies if I set my gnome to light theme, while it breaks in dark theme.
> 
> Renaming midnight/gtk-3.0/gtk.css to gtk-dark.css seems to fix it

Yes, adding 'gtk-dark.css' will fix it on Gnome's default dark theme settings.
Thanks!
